### PR TITLE
#18401: Implements the Drop Zone web component in WYSIWYG

### PR DESCRIPTION
**Pull request by ivorpad**

### CODE REVIEW

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field_js.jsp [hunk 1]
```diff
@@ -368,10 +375,30 @@ var cmsfile=null;
 			}else if(tinyConf.plugins != undefined ){
 			    tinyConf.plugins=tinyConf.plugins.replace("compat3x","");
 			}
-            console.log(textAreaId, tinyConf );
+            console.log(textAreaId, tinyConf);
```
Comment (alfredo-dotcms) [inline dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field_js.jsp:378]: remove

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp [hunk 2]
```diff
@@ -236,10 +236,22 @@
...
+                <style type="text/css">
+                   dot-asset-drop-zone {
+                     width: 100%;
+                     height: 91%;
+                     position: absolute;
+                     z-index: 1;
+                     pointer-events: none;
+                   }
+                </style>
+                <dot-asset-drop-zone id="dot-asset-drop-zone-<%=field.getVelocityVarName()%>"></dot-asset-drop-zone>
```
Comment (hmoreras) [inline dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:249]: why not the `dot-asset-drop-zone` wrap the textarea ?

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field_js.jsp (hunk 1 above)
Comment (hmoreras) [inline dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field_js.jsp:378]: remove

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp (hunk 2 above)
Comment (ivorpad) [reply dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp:249]: I understand this was its intended usage, however, when wrapping the textarea with it I had issues with pointer-events in the editor. If this is a problem please let me know.

Related https://github.com/dotCMS/core/blob/f6352f95de7842889b29490cca8246088ceedf13/dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field_js.jsp#L381-L393

**Comment by ivorpad** [comment]
Progress on https://github.com/dotCMS/core/issues/18401

**Review by freddyucv — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Pull request merged**